### ISSUE TITLE
Make default time at 00:00 instead of 12:00

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/wishlist#276]
## What is the need / problem?

Currently when using reports, the default time is 12:00 so users need to constantly change to 00:00 to not miss any info.

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/30d7eb6d-c4b2-4a8d-8ed5-8b4eae06d638)

## Which type of users does this problem affect (and how many, if known)?

_- Hub Managers (MCFEs)_
_- Producers_
_- Instance Managers_

## Potential solutions that will solve the problem  ?[[brainstorming to list feature candidates]

Put 00:00 (midnight) as a default value

**Comment by drummer83** [comment]
I totally agree for start dates. Should we make it 23:59 for end dates? :thinking:
